specification/ai/OpenAI.Assistants/client.tsp

```diff
@@ -10,60 +10,43 @@ namespace Azure.AI.OpenAI.Assistants;
 
 // Trivial response value containers should treat these containers as internal details and instead surface a shimmed
 // operation that exposes the underlying data directly.
-@@access(DeletionStatus, Access.internal);
-@@projectedName(DeletionStatus, "csharp", "InternalDeletionStatus");
 @@access(AssistantDeletionStatus, Access.internal);
 @@projectedName(AssistantDeletionStatus,
   "csharp",
   "InternalAssistantDeletionStatus"
 );
-@@access(Assistants.deleteAssistant, Access.internal);
-@@projectedName(Assistants.deleteAssistant,
-  "csharp",
-  "InternalDeleteAssistant"
-);
+@@access(deleteAssistant, Access.internal);
+@@projectedName(deleteAssistant, "csharp", "InternalDeleteAssistant");
 @@access(AssistantFileDeletionStatus, Access.internal);
 @@projectedName(AssistantFileDeletionStatus,
   "csharp",
   "InternalAssistantFileDeletionStatus"
 );
-@@access(Assistants.deleteAssistantFile, Access.internal);
-@@projectedName(Assistants.deleteAssistantFile,
-  "csharp",
-  "InternalUnlinkAssistantFile"
-);
+@@access(deleteAssistantFile, Access.internal);
+@@projectedName(deleteAssistantFile, "csharp", "InternalUnlinkAssistantFile");
 @@access(ThreadDeletionStatus, Access.internal);
-@@access(AssistantThreads.deleteThread, Access.internal);
-@@projectedName(AssistantThreads.deleteThread,
-  "csharp",
-  "InternalDeleteThread"
-);
-@@access(Assistants.listAssistants, Access.internal);
-@@projectedName(Assistants.listAssistants, "csharp", "InternalGetAssistants");
-@@access(Assistants.listAssistantFiles, Access.internal);
-@@projectedName(Assistants.listAssistantFiles,
-  "csharp",
-  "InternalGetAssistantFiles"
-);
-@@access(ThreadMessages.listMessageFiles, Access.internal);
-@@projectedName(ThreadMessages.listMessageFiles,
-  "csharp",
-  "InternalGetMessageFiles"
-);
-@@access(RunSteps.listRunSteps, Access.internal);
-@@projectedName(RunSteps.listRunSteps, "csharp", "InternalGetRunSteps");
-@@access(ThreadMessages.listMessages, Access.internal);
-@@projectedName(ThreadMessages.listMessages, "csharp", "InternalGetMessages");
-@@access(ThreadRuns.listRuns, Access.internal);
-@@projectedName(ThreadRuns.listRuns, "csharp", "InternalGetRuns");
+@@access(deleteThread, Access.internal);
+@@projectedName(deleteThread, "csharp", "InternalDeleteThread");
+@@access(listAssistants, Access.internal);
```

Comment:
@trrwilson Can we remove those @@access(deleteThread|listAssistants|..., Access.internal)?